packages/rocketchat-slackbridge/package.js

```diff
@@ -15,8 +15,13 @@ Package.onUse(function(api) {
 
 	api.addFiles('logger.js', 'server');
 	api.addFiles('settings.js', 'server');
+	api.addFiles('rocket.js', 'server');
+	api.addFiles('slack.js', 'server');
 	api.addFiles('slackbridge.js', 'server');
 	api.addFiles('slashcommand/slackbridge_import.server.js', 'server');
+
```

Comment:
Exporting the objects from Rocket and Slack JS files so they could see each other